### PR TITLE
chore(demo): `BlockStatus` add tuiSlot to api-page

### LINKED ISSUE #4376: 🐞 - Layout API issue
### Playground Link

https://taiga-ui.dev/layout/block-status/API

### Description

image

@vladimirpotekhin fyi

### Angular version

latest

### Taiga UI version

latest

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Documentation content changes

## What is the current behavior?

Closes #4376 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/components/block-status/block-status.template.html [hunk 1]
```diff
@@ -43,6 +43,7 @@
         <tui-doc-demo>
             <tui-block-status [card]="card">
                 <img
+                    tuiSlot="top"
```
Comment (splincode) [inline projects/demo/src/modules/components/block-status/block-status.template.html:46]: the `Setup` should before the `API` section

image

Comment (splincode) [inline projects/demo/src/modules/components/block-status/block-status.template.html:46]: Do we need to fix and add `tuiSlot` here?

image

**Comment by splincode** [comment]
please rollback lock file

File: projects/demo/src/modules/components/block-status/block-status.template.html (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/block-status/block-status.template.html:46]: why?

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/block-status/block-status.template.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/block-status/block-status.template.html:46]: oh, sorry, my fault

**Pull request merged**